### PR TITLE
demote gossip request counters logging to debug

### PR DESCRIPTION
**Pull request by HaoranYi**
#### Problem

https://github.com/solana-labs/solana/issues/24319
For rpc node, the metric agent is unable to catching up with logging at info level. gossip request counters are among the heaviest loggers.

#### Summary of Changes
demote gossip request counters logging to debug

Fixes #

### CODE REVIEW

**Comment by behzadnouri** [comment]
do the `..._verify_fail` ones actually show up in the logs?

**Comment by HaoranYi** [comment]
> do the `..._verify_fail` ones actually show up in the logs?

They don't show up. Only. *_sent and *_dropped are spaming.

File: core/src/window_service.rs [hunk 1]
```diff
@@ -132,7 +132,7 @@ impl ReceiveWindowStats {
             ("elapsed_micros", self.elapsed.as_micros(), i64),
         );
         for (slot, num_shreds) in &self.slots {
-            datapoint_info!(
+            datapoint_debug!(
                 "receive_window_num_slot_shreds",
                 ("slot", *slot, i64),
                 ("num_shreds", *num_shreds, i64)
```
Comment (behzadnouri) [inline core/src/window_service.rs:138]: cc @carllin 
see https://github.com/solana-labs/solana/issues/24319#issuecomment-1099179615

**Comment by behzadnouri** [comment]
https://github.com/solana-labs/solana/pull/24451 removes `inc_new_counter_...` ones.

**Comment by HaoranYi** [comment]
Closed. See https://github.com/solana-labs/solana/pull/24451 instead.

**Pull request closed without merging**